projects/observability/src/pages/explorer/explorer.component.test.ts

```diff
@@ -110,14 +118,15 @@ describe('Explorer component', () => {
     discardPeriodicTasks(); // Some of the newly instantiated components also uses async, need to wait for them to settle
   };
 
-  beforeEach(fakeAsync(() => {
-    spectator = createComponent();
+  const init = (...params: Parameters<typeof createComponent>) => {
```

Comment:
Why changing this to init function?

Reply:
This is generally why I prefer to keep the component creation outside of beforeEach and in each test instead - I had a use case in one of the tests where I needed to override the injects to test its initialization behavior (activated route in this case, I believe)